### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by rmrmrmr**
On this milestone we:
- Implemented React Routing: 
    - Added two pages to the website: one that acts as the homepage with a long text about math and one that includes a quote ➗➕
    - Added a navigation bar shared across the three pages of the website 🧭
- Added styles with React basic styling and TailwindCSS 🎨🖌️

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -15,33 +16,42 @@ const Calculator = () => {
...
+        <button className="orangeBttn" type="button" name="-" onClick={handleClick}>-</button>
+        <button className="greyBttn" type="button" name="1" onClick={handleClick}>1</button>
+        <button className="greyBttn" type="button" name="3" onClick={handleClick}>2</button>
+        <button className="greyBttn" type="button" name="3" onClick={handleClick}>3</button>
+        <button className="orangeBttn" type="button" name="+" onClick={handleClick}>+</button>
+        <button className="greyBttn w-1/2 rounded-bl-2xl" type="button" name="0" onClick={handleClick}>0</button>
+        <button className="greyBttn" type="button" name="." onClick={handleClick}>.</button>
+        <button className="orangeBttn rounded-br-2xl" type="button" name="=" onClick={handleClick}>=</button>
+      </div>
     </div>
```
Comment (GzimAsani) [inline src/components/Calculator.js]: - [ ] If the numbers are too long they should not go outside the calculator, you could use overflow hidden to achieve that
image

Comment (GzimAsani) [reply src/components/Calculator.js]: - [ ] When clicking number 2, the number 3 gets selected, kindly fix this typo https://www.loom.com/share/2c5622fd996d47a4b4ff11e1e4827877

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @rmrmrmr                    :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling, looks great :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @rmrmrmr,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**